framework/decode/openxr_replay_consumer_base.cpp

```diff
@@ -842,6 +842,25 @@ void OpenXrReplayConsumerBase::Process_xrReleaseSwapchainImage(
     CheckResult("xrReleaseSwapchainImage", returnValue, replay_result, call_info);
 }
 
+void OpenXrReplayConsumerBase::Process_xrEndFrame(const ApiCallInfo&                            call_info,
+                                                  XrResult                                      returnValue,
+                                                  format::HandleId                              session,
+                                                  StructPointerDecoder<Decoded_XrFrameEndInfo>* frameEndInfo)
+{
+    XrSession             in_session = MapHandle<OpenXrSessionInfo>(session, &CommonObjectInfoTable::GetXrSessionInfo);
+    const XrFrameEndInfo* in_frameEndInfo = frameEndInfo->GetPointer();
+    MapStructHandles(frameEndInfo->GetMetaStructPointer(), GetObjectInfoTable());
+
+    XrFrameEndInfo replay_frame_end_info = *in_frameEndInfo;
+    SessionData&   session_data          = GetSessionData(in_session);
+    replay_frame_end_info.displayTime    = session_data.GetDisplayTime();
```

Comment:
I would add a comment here about what you're doing.  It is easy to overlook this.